pages/api/availability/week.ts

```diff
@@ -0,0 +1,30 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
Codacy has a fix for the issue: [Replace `'next'` with `"next"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
import type { NextApiRequest, NextApiResponse } from "next";
```